openhtmltopdf-core/src/main/java/com/openhtmltopdf/render/BlockBox.java

```diff
@@ -364,7 +364,11 @@ public void createMarkerData(LayoutContext c) {
             imageMarker = result.getImageMarker() != null;
         }
 
-        if (listStyle != IdentValue.NONE && ! imageMarker) {
+        CascadedStyle markerStyle = c.getCss().getPseudoElementStyle(_element,"marker");
```

Comment:
Is this `"marker"` string stored anywhere? Would be preferable to get this from a `CSSName` or some similar enum rather than injecting a "magic value" here.

Reply:
At a quick glance it looks like we could replace with `IdentValue.MARKER.toString()` throughout.